tests/core/string/test_node_path.h

```diff
@@ -167,6 +167,44 @@ TEST_CASE("[NodePath] Empty path") {
 			node_path_empty.is_empty(),
 			"The node path should be considered empty.");
 }
+
+TEST_CASE("[NodePath] Slice") {
```

Comment:
Your test cases don't include any absolute paths. You can copy the tests from #78638 if you want.

Reply:
Good idea. 

I don't know that it's necessary to double every test case so that there is a relative version and an absolute version as the slicing logic is identical in both cases.

I'll push some extra test cases today.